Vagrantfile

```diff
@@ -11,7 +11,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     c.vm.box = "trusty64"
     c.vm.box_url = "https://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-i386-vagrant-disk1.box"
 
-    c.vm.provision :shell, inline: "apt-get update"
+    c.vm.provision :shell, :path => "bootstrap.sh"
     c.vm.provision :puppet do |puppet|
       puppet.module_path = "modules"
```

Comment:
I think we need to create the modules directory and store the empty directory in git for the puppet provisioner. Does it work on your machine without having the modules directory?